Gemfile

```diff
@@ -107,3 +107,7 @@ end
 group :development, :test do
   gem 'fcrepo_wrapper'
 end
+
+group :profile do
+  gem 'ruby-prof'
```

Comment:
can we include ruby-prof in dev and test too? Any downsides?  I guess we could do it with `require: false` just to make _sure_ it's having no effect at all in those envs (would have to manually `require` it if you wanted to use it). Don't know if tha'ts neccesary, it ought to have no effect if not being used, and it's not in production.